### ISSUE TITLE
SHANK3 'duplicated' annotation

### ISSUE THREAD

**Issue by selewis** [geneontology/go-annotation#2019]
AKA Q9BYB0 or HGNC:14294
It has 3 annotations to plasma membrane (GO:0005886). One an ISS from BCL and two from PAINT (IBAs). The only difference between the two IBA annotations is that one is qualified as 'colocalizes_with' and the other does not. Seems like we should only have one or the other. If we're talking about a cellular location does that imply/mean we're saying colocalizes with already? Or do we need to call this out explicitly each time in the annotation. @pgaudet perhaps you could rule on this (and eliminate the extra annotation)

**Comment by RLovering** [comment]
Hi 
I have just checked this and changed the annotation to GO:0031234 extrinsic component of cytoplasmic side of plasma membrane in case this more specific annotation is of use.

Ruth

**Comment by pgaudet** [comment]
@selewis I agree something PAINT shouldn't allow two annotations that differ by the qualifier (what do you think @huaiyumi  @marcfeuermann @thomaspd ) ?

**Comment by marcfeuermann** [comment]
Yes, in my opinion it makes no sens to propagate two annotations that differ only by the qualifier. One should chose the most relevant, knowing the general context.
Regards,
Marc.

**Comment by thomaspd** [comment]
Yes, that makes sense.  @marcfeuermann , do you have time to go in and fix this specific annotation manually in PAINT?  @mugitty, do you have time to add a check in PAINT to make sure the same annotation can only be made once, even if the qualifier is different?

**Comment by mugitty** [comment]
The paint software will not permit 2 annotations to the same node with different qualifiers. Some other process has added these.  Furthermore, the IBA annotations will only be propagated to leaf nodes not providing evidence for the annotation.

You can test by doing the following on the latest version of the software ( Paint Version 2018_06_08_1)
1. Open book PTHR24135
2. Left click on node Metozoa-Chonaoflagellida (PTN001191878).  This happens to be the root node.
3. Bring up the annotations panel by Clicking on the Annotations tab.
4. Delete annotations to plasma membrane  GO:0005886 (delete both annotations), ionotropic glutamate receptor complex - GO:0008328, and ionotropic glutamate receptor binding - GO:0035255. Note, Annotations to GO:0008328 and GO:0035255 have to be deleted since these annotations are more specific than GO:0005886.
5. Select the Cellular Component tab.  This will bring up the cellular component annotations on the annotations matrix.
6. Drag plasma membrane annotation from annotation matrix to node Metozoa-Chonaoflagellida (PTN001191878)
7. The system will bring up a dialog box prompting the user to select the applicable qualifiers and evidence nodes
![image](https://user-images.githubusercontent.com/6527204/43599504-af35c7a0-963c-11e8-8fcb-079a890c0a9b.png)
8. Depending on what qualifiers are chosen above, the system will only use those nodes and qualifiers as evidence for the annotation.  You can click on the descendants and verify.  Note, the nodes providing the evidence will not get the IBA annotation

**Comment by pgaudet** [comment]
Hi @mugitty 

Whatever is the cause if this, is also seen in Pantree: 

<img width="757" alt="image" src="https://user-images.githubusercontent.com/4782928/43601158-e6986dec-968c-11e8-95a5-319c255b4c47.png">



I must say I completely dont understand the data flow, and I have been unable to pinpoint the cause of many various errors. 

Thanks ! 

Pascale

**Comment by mugitty** [comment]
The software will not allow the curator to annotate to the same term twice.  Something in the load, etc has added the extra annotation.

**Comment by pgaudet** [comment]
Looking at the PAINT family annotations with @huaiyumi 
this does look like a bug. Perhaps it's not possible to that this is caused by family merges, although its not possible to do it in PAINT directly ? 

In any case there is a real problem that we can also see in PAINT.

**Comment by pgaudet** [comment]
Not resolved in the Aug 7-2018 PAINT files: ftp://ftp.pantherdb.org/downloads/paint/presubmission/gene_association.paint_human.gaf.gz

  UniProtKB | Q9BYB0 | SHANK3 | COLOCALIZES_WITH | GO:0005886 | PMID:21873635 | IBA | PANTHER:PTN001191878\|MGI:MGI:1930016\|MGI:MGI:3613677\|MGI:MGI:2671987\|UniProtKB:Q9Y566 | C | SH3 and multiple ankyrin repeat domains   protein 3 | UniProtKB:Q9BYB0\|PTN002516415 | protein | taxon:9606 | 21.04.17 | GO_Central
  UniProtKB | Q9BYB0 | SHANK3 |   | GO:0005886 | PMID:21873635 | IBA | PANTHER:PTN001191878\|MGI:MGI:1930016\|MGI:MGI:3613677\|MGI:MGI:2671987\|UniProtKB:Q9Y566 | C | SH3 and multiple ankyrin repeat domains protein 3 | UniProtKB:Q9BYB0\|PTN002516415 | protein | taxon:9606 | 22.09.17 | GO_Central

**Comment by pgaudet** [comment]
'Source ticket: https://github.com/pantherdb/fullgo_paint_update/issues/8

**Comment by pgaudet** [comment]
Probably fixed ? check at next release (when human IBAs are back in AmiGO)

**Comment by ValWood** [comment]
fixed!
